services/messagebus.md

```diff
@@ -0,0 +1,23 @@
+# Message Bus
+
+    identifier: 0x17feef9b
+    
+A lightweight event bus that allows to broadcast
+short messages.
+
+## Commands
+
+    command send @ 1 {
```

Comment:
this probably needs to be @ 0x80 - you should be getting an error here?

Reply:
will check